pyuvdata/uvdata.py

```diff
@@ -366,11 +367,23 @@ def check(self, check_extra=True, run_check_acceptability=True):
         Check that required parameters exist. Check that parameters have
         appropriate shapes and optionally that the values are acceptable.
 
-        Args:
-            check_extra: If true, check all parameters, otherwise only check
-                required parameters.
-            run_check_acceptability: Option to check if values in parameters
-                are acceptable. Default is True.
+        Parameters
+        ----------
+        check_extra : bool
+            If true, check all parameters, otherwise only check required parameters.
+        run_check_acceptability : bool
+            Option to check if values in parameters are acceptable.
+
+        Returns
+        -------
+        bool
+            True if check passes
```

Comment:
This rendering looks funny to me:
![image](https://user-images.githubusercontent.com/4752917/58767685-05051400-8544-11e9-97e7-1f085e818dce.png)
I'm not sure if that's the style we are going for or not. For example, the numpy docs usually specify a variable name (even if it's irrelevant to the user), and the structure looks the same as for arguments.
e.g.:
![image](https://user-images.githubusercontent.com/4752917/58767709-50b7bd80-8544-11e9-96e9-51d5b05d1810.png)